images/nginx/rootfs/build.sh

```diff
@@ -106,6 +106,9 @@ export OPENTELEMETRY_CPP_VERSION="v1.11.0"
 # Check on https://github.com/open-telemetry/opentelemetry-proto
 export OPENTELEMETRY_PROTO_VERSION="v1.1.0"
 
+# http://hg.nginx.org/njs
+export NGINX_NJS_VERSION="0.8.4"
```

Comment:
Latest is 0.8.7

Reply:
The latest one is not available at https://hg.nginx.org/njs/tags. However, if we want the latest we can build it from [source](https://github.com/nginx/njs/tree/0.8.7).

Reply:
> The latest one is not available at https://hg.nginx.org/njs/tags. However, if we want the latest we can build it from [source](https://github.com/nginx/njs/tree/0.8.7).

The official NJS repository has been migrated to GitHub earlier this year. hg.nginx.org is a read-only mirror now, and due to a difference in tagging implementations it was not possible to synchronize the tags to Mercurial.

All recent releases are available at https://github.com/nginx/njs/tags, and the download URL could be changed to `https://github.com/nginx/njs/archive/${NGINX_NJS_VERSION}.tar.gz`.

Reply:
Done.
Thanks @bavshin-f5.